src/plugins/matching-cards/matching-cards-display.js

```diff
@@ -100,8 +100,8 @@ function MatchingCardsDisplay({ content }) {
           <MatchingCardsTile
             text={tile.text}
             sourceUrl={tile.sourceUrl}
-            playMedia={isFlipped}
             showMatched={wasMatched}
+            playMedia={isFlipped && !wasMatched}
```

Comment:
When 2 cards are matched we show the grey overlay and green checkmark, but the media still plays underneath. I think it's better that we stop it in this case. What do you think?

Reply:
Completely agree.